csm_web/frontend/static/frontend/css/style.css

```diff
@@ -232,6 +232,13 @@ header {
 	font-weight: bold;
 }
 
+.course-card-section-time:not(:last-of-type)::after {
+	display: block;
+	font-weight: bold;
+	content: "&";
```

Comment:
nitpick: bolding the ampersand feels like it's trying to emphasize the wrong thing, if anything I think it'd be better to bold the spacetimes